requirements/common.txt

```diff
@@ -6,10 +6,11 @@ gunicorn~=20.1.0
 openpyxl~=3.0.9
 python-dotenv~=0.19.2
 python-magic~=0.4.24
-gemmi~=0.5.7
+gemmi~=0.6.6
 xlrd~=2.0.1
 Werkzeug~=2.2.2
 jcamp~=1.2.2
 PyMuPDF==1.23.7
 pylint==3.0.3
-str2bool~=1.1
\ No newline at end of file
+str2bool~=1.1
+parser-binary @ git+https://github.com/ComPlat/BinaryParser@main
```

Comment:
does this install the binary artifact from the repo or does this trigger the building from the source ?

Reply:
What dependencies are needed at runtime ? Does the Readme needs an update